### PR TITLE
Keysight win

### PR DESCRIPTION
**Pull request by DavidHVernon**

### CODE REVIEW

File: Headers/GNUstepBase/GSConfig.h.in [hunk 1]
```diff
@@ -299,7 +299,7 @@ typedef	struct {
 #endif
 
 #if defined(_WIN32)
-#  define BOOL WinBOOL
```
Comment (triplef) [inline Headers/GNUstepBase/GSConfig.h.in:302]: Pretty sure `WinBOOL` is correct, at least under MinGW64 and MSVC.

The MSVC CI build is failing with `WINBOOL`:
> error: unknown type name 'WinBOOL'; did you mean 'WINBOOL'?

Comment (DavidHVernon) [reply Headers/GNUstepBase/GSConfig.h.in:302]: Interesting. The error I get on mingw64 is the exact opposite: 

`error: unknown type name 'WINBOOL'; did you mean 'WinBOOL'?
 `

Comment (triplef) [reply Headers/GNUstepBase/GSConfig.h.in:302]: Works fine under MinGW64 with and without your change on CI though – wonder what’s different in your setup?
https://github.com/gnustep/libs-base/runs/5688864693?check_suite_focus=true

File: Headers/GNUstepBase/GSConfig.h.in [hunk 2]
```diff
@@ -299,7 +299,7 @@ typedef	struct {
...
+#  define BOOL WINBOOL
```
Comment (gcasa) [inline Headers/GNUstepBase/GSConfig.h.in:302]: @triplef Did you mean to comment here, for this change?

**Comment by triplef** [comment]
It would be good to understand what this is trying to fix/improve. If it's still needed it should be rebased to run on the current CI targets.

**Comment by gcasa** [comment]
@DavidHVernon discussed with OP.  @rfm I am going to close this issue as I don't think it is currently a problem.  @triplef

**Pull request closed without merging**